editor/js/libs/ui.js

```diff
@@ -1130,6 +1130,7 @@ class UITabbedPanel extends UIDiv {
 		if ( tab ) {
 
 			tab.addClass( 'selected' );
+			tab.dom.scrollIntoView( { inline: 'center', behavior: 'smooth' } );
```

Comment:
That's nice! I didn't know about this 🙏

Reply:
congrats 🎉

Reply:
Note that there's an issue when using `scrollIntoView` in the editor, and its solved by using `scrollTo` instead, see #28485 🙏🏻 sorry for that.